classes/article/ArticleDAO.inc.php

```diff
@@ -180,23 +180,26 @@ function updateObject($article) {
 				(int) $article->getStageId(),
 				$article->getLanguage(),
 				$article->getCommentsToEditor(),
-				$article->getCitations(),
 				(int) $article->getStatus(),
 				(int) $article->getSubmissionProgress(),
 				(int) $article->getCurrentRound(),
 				$article->getPages(),
 				(int) $article->getFastTracked(),
 				(int) $article->getHideAuthor(),
+				(int) $article->getHideSubmissionRevisions(),
 				(int) $article->getId()
 			)
 		);
 
 		$this->updateLocaleFields($article);
-
+		$contextId = Request::getContext()->getId();
+		$version = $this->getLatestRevisionId($article->getId(), $contextId);
```

Comment:
Would it be possible to get the version/revisionId from article object (that is being updated) itself?